source/github-events/on-diff-file-load.ts

```diff
@@ -11,9 +11,13 @@ const getDeduplicatedHandler = mem((callback: EventListener): delegate.EventHand
 	event.delegateTarget.addEventListener('load', callback);
 });
 
-export default function onDiffFileLoad(callback: EventListener): delegate.Subscription {
+export function createFragmentLoadListener(fragmentSelector: string, callback: EventListener): delegate.Subscription {
```

Comment:
Ohhh there it was. Glad you found it. The file needs to be renamed though. It can export both functions without a default.